src/raftstore/store/peer.rs

```diff
@@ -1601,8 +1601,14 @@ impl Peer {
             return Ok(ctx);
         }
 
-        if req.get_admin_request().has_split() {
-            ctx.insert(ProposalContext::SPLIT);
+        match req.get_admin_request().get_cmd_type() {
+            AdminCmdType::Split => {
+                return Err(box_err!(
```

Comment:
What if a new store send such a proposal to an old store during rolling update?

Reply:
Sorry. What I means is an old store send such a proposal to a new store.

Reply:
Proposals are encoded as log entry instead.